src/routes/docs/tooling/keyboard/+page.markdoc

```diff
@@ -0,0 +1,39 @@
+---
+layout: article
+title: Keyboard
+description: How to use the keyboard with the optimized Appwrite Console for both efficiency and accessibility, and introducing The Appwriter.
+---
+
+The Appwrite SDK console is optimized with the keyboard using shortcut keys to enable accessibility.  Using shortcut keys access websites without the mouse. 
+
+|Keys|Action|
+|---|---|
+|`Ctrl` + `K` or `⌘` + `K`| access command center|
+|`ENTER`|execute command center command|
+
```

Comment:
I don't think we need to wait for Elad to add an a section about accessibility, we can adjust later if needed, here is an example of how it might look like:

```
We are committed to making the Appwrite Console accessible to all developers. For that, the platform supports the AA standard of the Web Content Accessibility Guidelines (WCAG). This standard ensures that our dashboard is usable by people with a wide range of disabilities, including visual, auditory, physical, speech, cognitive, language, learning, and neurological disabilities.

Supporting the AA standard means our dashboard provides good contrast between text and backgrounds, resizable text without loss of content or functionality, and navigable interfaces via keyboard and screen readers. These features are crucial for creating an inclusive environment where all users can perform their tasks efficiently. By adhering to these guidelines, we ensure that our dashboard not only complies with legal requirements but also promotes a positive developer experience for everyone.
```